src/fideslang/models.py

```diff
@@ -823,11 +912,12 @@ class PrivacyDeclaration(BaseModel):
     data_use: FidesKey = Field(
         description="The Data Use describing a system in a privacy declaration.",
     )
-    data_qualifier: FidesKey = Field(
+    data_qualifier: Optional[FidesKey] = Field(
         default="aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified",
-        description="The fides key of the data qualifier describing a system in a privacy declaration.",
+        description="Deprecated. The fides key of the data qualifier describing a system in a privacy declaration.",
     )
     data_subjects: List[FidesKey] = Field(
+        default=[],
```

Comment:
Same suggestion as @ThomasLaPiana below

```suggestion
        default_factory=list,
```

Reply:
thanks responded above! https://github.com/ethyca/fideslang/pull/146#discussion_r1290219668